src/main-process/main.js

```diff
@@ -129,6 +136,26 @@ app.on('ready', () => {
   // integrations
   const integrations = new IntegrationManager(mainWindow);
   integrations.enable();
+
+  // Update tray icon for gnome-shell
+  if(process.platform === 'linux') {
+    const ps = require('ps-node');
+    // detect gnome-shell process
+    ps.lookup({
+      command: 'gnome-shell',
+    }, (err, resultList ) => {

```

Comment:
shouldn't linter catch this weird spacing ?

Reply:
nope, checkout npm run lint:js don't see any issues with this spacings